meta-signing-key/recipes-support/key-store/key-store_0.1.bb

```diff
@@ -37,7 +37,7 @@ SECONDARY_TRUSTED_DER_ENC_CERT = "${KEY_DIR}/x509_secondary_system_trusted_key.d
 MODSIGN_CERT = "${KEY_DIR}/modsign_key.crt"
 
 # For ${PN}-ima-cert
-IMA_CERT = "${KEY_DIR}/x509_evm.der"
```

Comment:
Both initrdscripts/files/ and linux-yocto/ima.cfg assumes to use this name pattern. Plz modify them together and have a running test.

Reply:
Yes, init.ima and kernel cfg need  modification as well.